tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/ml/Verbnet.scala

```diff
@@ -14,11 +16,16 @@ import DefaultJsonProtocol._
   * (http://projects.csail.mit.edu/jverbnet/),  to quickly look up various vernbet
   * features for a verb.
   */
-case class Verbnet(verbnetPath: String) {
+case class Verbnet(groupName: String, artifactName: String, version: Int) {
 
   // Construct the index and open it
   @transient val index = {
-    val url = new URL("file", null, verbnetPath)
+    val verbnetPath: java.nio.file.Path = Datastore.directoryPath(
```

Comment:
I belief there's a call that can get a file URL directly.

Also, you should leave off the type declaration - it's just for documenting the return type in Dirk's example code.